### ISSUE TITLE
Issue with aria-labelled by associated with summary element

### ISSUE THREAD

**Issue by chmnata** [CityofToronto/bdit_cot-vfh#55]
Issue

The `<table>` elements there is an aria-labelledby attribute that is bound to the <summary> element.  Ideally when a user scans the page in browse mode the aria-labelledby reference would be announce by the screen reader.  However, the accessible name of the data table is not announced.  Note there have been inconsistent browser/AT support for the aria-labelledby attribute for the `<table> `element.  Also, the word "table" is used in the  <summary> element which means if the aria-labelledby worked correctly, "table" would be repeated.

Recommendation

Use caption element to provide an accessible name for a data table.

**Comment by chmnata** [comment]
Kevin suggested changing the word `Table` to `Raw Data` in the title because table will be repeated twice when using a screen reader. Would `Summary of average number of trips per hour, February 2020` makes more sense? Since its not really raw data. 🤔 
![image](https://user-images.githubusercontent.com/46324452/150168793-547c097a-163f-4ce7-9c5a-1285b6294505.png)
